### PR TITLE
Adding CS and github actions

### PR DESCRIPTION
**Pull request by Nyholm**
I hope the Rove BC checker works on this repo =)

### CODE REVIEW

File: .github/workflows/static.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: PHPStan
+        uses: docker://oskarstark/phpstan-ga
+        with:
+          args: analyze --no-progress
```
Comment (rvanlaak) [inline .github/workflows/static.yml]:
```suggestion
          args: analyze . --no-progress
```

**Comment by rvanlaak** [comment]
What about adding CS-Fixer and phpstan in different PRs?

File: .github/workflows/static.yml (hunk 1 above)
Comment (Nyholm) [reply .github/workflows/static.yml]: Oops, You are correct. I missed the config file.

**Comment by Nyholm** [comment]
Sure: #331

**Comment by Nyholm** [comment]
This is only changes to meta files, added Roave BC check and php-cs-fixer. 

I also fixed some CS issues.

**Comment by Nyholm** [comment]
@rvanlaak By the same logic, should we merge this and hope for the best?
=)

**Comment by rvanlaak** [comment]
Somehow they did do run once https://github.com/php-translation/symfony-bundle/commit/ae1c6b07b252f14a25f1646c7deb36b9b8707121/checks

But yes, let's try merging, as this PR has cs fixes as well.

**Pull request merged**